Jenkinsfile.groovy

```diff
@@ -102,7 +93,28 @@ pipeline {
 			}
 		}
 	}
+	post {
+		success {
+			setBuildStatus("Build successful", "SUCCESS", "${STAUS_CONTEXT}", "${GITHUB_BRANCH_HEAD_SHA}")
+		}
+		failure {
+			setBuildStatus("Build failed", "FAILURE", "${STAUS_CONTEXT}", "${GITHUB_BRANCH_HEAD_SHA}")
+		}
+	}
 }
+
+def setBuildStatus(String message, String state, String context, String sha) {
+    step([
+        $class: "GitHubCommitStatusSetter",
+        reposSource: [$class: "ManuallyEnteredRepositorySource", url: "https://github.com/Metrilus/MetriCam2"],
+        contextSource: [$class: "ManuallyEnteredCommitContextSource", context: context],
+        errorHandlers: [[$class: "ChangingBuildStatusErrorHandler", result: "UNSTABLE"]],
+        commitShaSource: [$class: "ManuallyEnteredShaSource", sha: sha ],
+        statusBackrefSource: [$class: "ManuallyEnteredBackrefSource", backref: "${BUILD_URL}flowGraphTable/"],
```

Comment:
We should sanitize the BUILD_URL, or try jenkins to fill a proper value in there.
-> goes into issue